guides/concepts/networks.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Networks
+---
+
+Stellar has two public networks: the Public Network (pubnet), which is the main decentralized
+network used throughout the world, and the Test Network (testnet), a network maintained by the
```

Comment:
Since both networks are used throughout the world, I'd make a different distinction, and keep the construction parallel:

"Stellar has two public networks: the Public Network (pubnet), which is the main network used for applications in production, and the Test Network (testnet), which is a network maintained by the Stellar Development Foundation that developers can use to test their applications.